src/main/java/frc/robot/commands/Turret/VisionAimTurret.java

```diff
@@ -51,28 +51,30 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    limelightTarget = -vision.limelight.getOffsetX() + turret.getTurretAngle();
+    if (RobotContainer.switchBoard.btn_9.get()) {
```

Comment:
check the switch in `RobotContainer.java` instead of the command. Ideally we don't read switches outside of `RobotContainer` just so it's easier to see when they're being used.